src/gridstack.js

```diff
@@ -1278,8 +1278,9 @@
             self.grid.cleanNodes();
             self.grid.beginUpdate(node);
             cellWidth = self.cellWidth();
-            var strictCellHeight = Math.ceil(o.outerHeight() / o.attr('data-gs-height'));
-            cellHeight = self.container.height() / parseInt(self.container.attr('data-gs-current-height'));
+            height = o.attr('data-gs-height');
+            verticalMargin = self.opts.verticalMargin;
+            cellHeight = Math.ceil((o.outerHeight() - (height - 1) * verticalMargin) / height);
```

Comment:
if I make this `var cellHeight` (like strictCellHeight was) then we get an infinite loop stack error between moveNode() and _prepareNode() !
Not sure why... I don't think it's calling the method and we do things like this in other places:
var cellHeight = self.cellHeight();